pkg/generators/openapi.go

```diff
@@ -440,7 +452,7 @@ func (g openAPITypeWriter) generateExtensions(CommentLines []string) error {
 		}
 	}
 	// Next, generate extensions from "tags".
-	for tagKey := range tagToExtension {
+	for _, tagKey := range sortedMapKeys(tagToExtension) {
```

Comment:
This is going to sort by tag, not extension. 

I'm not trying to make a point, I'm just making sure it's obvious :-)